ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdated.java

```diff
@@ -277,12 +278,22 @@ private JsonRpcResponse handleNonValidForkchoiceUpdate(
   }
 
   private void logPayload(final EnginePayloadAttributesParameter payloadAttributes) {
-    LOG.atDebug()
-        .setMessage("timestamp: {}, prevRandao: {}, suggestedFeeRecipient: {}")
-        .addArgument(payloadAttributes::getTimestamp)
-        .addArgument(() -> payloadAttributes.getPrevRandao().toHexString())
-        .addArgument(() -> payloadAttributes.getSuggestedFeeRecipient().toHexString())
-        .log();
+    LoggingEventBuilder loggingEventBuilder =
+        LOG.atDebug()
+            .setMessage(
+                "payloadAttributes: timestamp: {}, prevRandao: {}, suggestedFeeRecipient: {}")
```

Comment:
Do we want to add withdrawals to the message as well?

Reply:
done